docs/getting-started.md

```diff
@@ -0,0 +1,104 @@
+    2. Fulfillment centers
+    3. Operational timezone
+5. Setup theme
+
+Open your first store in a browser.
+
+The [Managing stores](https://github.com/VirtoCommerce/vc-content/blob/deploy/pages/docs/vc2userguide/configuration/store.md) article helps you to configure your first store. Also at this step you should configure [Users, Roles and Role Assignment](https://github.com/VirtoCommerce/vc-content/blob/deploy/pages/docs/vc2userguide/users-management-roles-and-role-assignment.md)
+
+## Step 3. Configure prices and inventory
+
+In the Virto Commerce, a pricelist is a set of pricing details that can be assigned to items. All prices are stored in a pricelist(s), which have specific currency set. When a pricelist is created, you define which products will be included into this pricelist and what prices will be defined for them.
+
+The way to apply the prices to products on Storefront is creating a Pricelist Assignment. It binds a catalog with pricelist and can add optional conditions for this binding.
+
+Pricelist assignment identifies the pricelist from which a product price is shown for customers. Moreover, pricelist assignments can do much more than simply provide default list and sale prices for products. This feature allows you to provide custom prices to targeted customers according to customer's characteristics such as age, gender, geographic location and searched terms.
+
+Thus, price configuring consists of the following actions:
+
+1. Create pricelist
+2. Add items to the pricelist
+
+3. Create pricelist assignment
+4. Manage inventory
+
+After this step completed, you can [Open new products in Virto Commerce Storefront](https://github.com/VirtoCommerce/vc-content/blob/deploy/pages/docs/lessons/lesson2.md#open-new-products-in-virto-commerce-storefront-frontend) , browse the catalog, add product to shopping cart and create order. You can view customer order via admin UI.
+
+## Step 4. Configure payments, shipments, taxes
+
+By default, you see demo payment and shipment methods. Virto Commerce provides the ready to use module which you can set your own configurations:
+
+* Install Payment module (For example: PayPal)
+* Configure [Shipments](https://github.com/VirtoCommerce/vc-content/blob/deploy/pages/docs/vc2userguide/order-management/working-with-shipments.md) method
+* Configure [Taxes](https://github.com/VirtoCommerce/vc-content/blob/deploy/pages/docs/vc2userguide/order-management/working-with-taxes.md) method
+
+## Step 5. Launch Marketing & promotional tools
+
+Marketing your store is essential to gain your customers loyalty and raise awareness among potential customers.
+
+There are several marketing tools in Virto Commerce: you can publish content, create personalized shopping experiences for customers, manage promotions.
+
+Start with creation of two simple promotions:
+
+1. Each customer would receive a 5%  off if cart subtotal exceeded $100
+2. Add Gift product to shopping cart for every fifth customer order
```

Comment:
interesting, but i don't think we currently support scenario like this

Reply:
Could it be `Add Gift product to shopping cart for a limited period of time` ?

Reply:
Fixed